data/learning.yml

```diff
@@ -276,6 +276,11 @@ books:
       - link: /learn/intrinsics/UNPACK
       - link: /learn/intrinsics/VERIFY
 
+  - title: Fortran Object Oriented Programming
+    description: Putting Fortran’s object-related features to practical use
+    category: Fortran Documentation
+    link: /learn/oop_features_in_fortran
```

Comment:
To link the sidebar, you would have to modify this to: 
```
pages:
      - link: /learn/oop_features_in_fortran
 ```

Reply:
@henilp105 Thank you for the valuable hint.  Based on commit [9e9b663](https://github.com/fortran-lang/webpage/pull/477/commits/9e9b663ff35d94cea85e358d6772648140c4df85), I launched a test build, and the booklet now equally is accessible from the side pane (provided an other booklet already was opened).

For an easier navigation, I now consider to edit the one-page booklet into a multi-page booklet.